### ISSUE TITLE
Issues with Tables v0.1.14

### ISSUE THREAD

**Issue by Alexander-Barth** [jverzani/Mustache.jl#91]
With Mustache v0.5.8 and Tables v0.1.12 I obtain the following result which I think is correct:

```
julia>  using Mustache; Mustache.render("{{#list}}{{ item }} {{/list}}",Dict("list"  => Any[Dict("item" => "one"),Dict("item" => "two")]))
"one two "
```

However, with Tables v0.1.14 (and still  Mustache v0.5.8), I get:
```
julia> using Mustache; Mustache.render("{{#list}}{{ item }} {{/list}}",Dict("list"  => Any[Dict("item" => "one"),Dict("item" => "two")]))
"  "
```
The `Any` is important to trigger this different behaviour.
I use Julia 1.0.1 on Linux, see the same behaviour on another machine with Julia 1.0.3.

**Comment by jverzani** [comment]
Thanks for letting me know about this. This update to `Tables.jl` broke 4 tests in addition to your test case! I'll see what I can do.

**Comment by jverzani** [comment]
Closing. Thanks again

**Comment by Alexander-Barth** [comment]
I am afraid, that that this issue is there again:

```
using Mustache; Mustache.render("{{#list}}{{ item }} {{/list}}",Dict("list"  => Any[Dict("item" => "one"),Dict("item" => "two")]))
```

returns now again an empty string. I am using the following version and Julia 1.0.1 on Linux.

```
  [ffc61752] Mustache v0.5.10
  [bd369af6] Tables v0.1.15
```

Is the committed fix for a released version of Tables?

**Comment by jverzani** [comment]
Thanks again! Hmm, yes, the fix was keeping up with Table v0.1.14. Let me try and upgrade to v0.1.15 and hope the new behaviour works as it had prior to v0.1.14, which forced a change.

**Comment by Alexander-Barth** [comment]
Thank you very much! It works also for me (Mustache v0.5.11, Tables v0.1.15).
Thanks for your great work!
